l3build.lua

```diff
@@ -28,31 +28,31 @@ for those people who are interested.
 release_date = "2020-06-04"
 
 -- File operations are aided by the LuaFileSystem module
+-- Next might be unnecessary on modern `texlua`
 local lfs = require("lfs")
 
 -- Local access to functions
 
-local assert           = assert
 local ipairs           = ipairs
 local insert           = table.insert
 local lookup           = kpse.lookup
 local match            = string.match
 local gsub             = string.gsub
-local next             = next
 local print            = print
-local select           = select
-local tonumber         = tonumber
 local exit             = os.exit
 
 -- l3build setup and functions
 kpse.set_program_name("kpsewhich")
-build_kpse_path = match(lookup("l3build.lua"),"(.*[/])")
+local build_kpse_path = match(lookup("l3build.lua"),"(.*[/])")
 local function build_require(s)
-  require(lookup("l3build-"..s..".lua", { path = build_kpse_path } ) )
+  return require(lookup("l3build-"..s..".lua", { path = build_kpse_path } ) )
 end
 
 -- Minimal code to do basic checks
-build_require("arguments")
+A = build_require("arguments")
```

Comment:
Even if we would use `A` inside of `arguments`, we shouldn't use such a generic and short name for a global.